### DISCUSSION TITLE
Scene Viewer stopped working on Samsung Note 10+

### DISCUSSION THREAD

**Discussion by SGECZ** [google/model-viewer#2255] [Q&A] [answered]
I know - it's not model viewer directly - but anyone has an idea what that could be?

Since today (April 4th, 2021) scene viewer stopped working on my Samsung Note 10+
I can see the model in model viewer, when it's handed over to scene viewer, it doesn't show up in AR.
Other Android phones (with the same Chrome/Google/ARCore version) work fine.

Anyone know where I can submit this directly to someone who is working on the code of this?

Thank you!

**Comment by SGECZ** [comment]
It is getting more weird - it might depend on the size of the model loaded and/or how fast scene viewer gets the data.

Maybe anyone can try this: https://cldxr.to/bgji6olz  - that model has 8,5 mb.

This works on the Note 10 with 4g - it does not work on Wifi.
On two different Samsung Note 10+ - exactly the same behavior - in different locations with different 4g provider

But on a Samsung S9 and other Androids it works fine on Wifi, even with larger files.

But when it fails on the Note 10, just for a milisecond that white circle shows up.
![almost](https://user-images.githubusercontent.com/68609303/113592343-51847380-9635-11eb-8c8d-d393ff5d01ef.JPG)

**Reply by elalish** [reply]
@tpsiaki @AdrianAtGoogle Looks like a Scene Viewer regression?

**Reply by SGECZ** [reply]
> 
> 
> @tpsiaki @AdrianAtGoogle Looks like a Scene Viewer regression?

Anyone have an idea? Or a contact? Anything? By now I know it's not related to Android 10 or 11, it's not related to the brand of the phone and it's not related to 4G or Wifi.

It's simply sceneviewer timing out or so when trying to open the model. Any help is appreciated, I am getting a bit desperate here. Thank you!

**Comment by SGECZ** [comment]
I am happy to give more info if needed. We are currently trying to find more Android 11 user - because so far all the devices where it works are Android 10 - and only the two Samsung Note with the issue are Android 11.
So if you have Android 11 please try this - with Wifi and 4G - and let me know if it worked: https://cldxr.to/bgji6olz 
Your help is much appreciated!
Thank you!

**Reply by SGECZ** [reply]
And by now we also found Huawei and Xiaomi user with Android 11 having a that issue - so not isolated to Samsung or the Note 10 - but it looks like it's Android 11

**Comment by david-rhodes** [comment, marked as answer]
Based on [this issue](https://github.com/google/model-viewer/issues/2264) it's a known issue. I am experiencing the same thing and I'm confused about whether or not this was considered fixed, yet. Any clarification in this matter would be appreciated. 

Thank you!

**Reply by elalish** [reply]
@tpsiaki Has this fix rolled out yet? Can we give a version number for them to check?

**Reply by SGECZ** [reply]
> 
> 
> Based on [this issue](https://github.com/google/model-viewer/issues/2264) it's a known issue. I am experiencing the same thing and I'm confused about whether or not this was considered fixed, yet. Any clarification in this matter would be appreciated.
> 
> Thank you!

Thank you, finally I know what's wrong here. I'd give you a hug if I could.

Not sure now if I should be happy to finally know or upset because this could have been answered much earlier. Also the email I got from someone at Google was never followed up after I replied and described the problem in detail.

I wonder what someone has to do as a developer to be taken seriously by a company that makes billions (I'm a shareholder!) but treats the people who help them spread their products and technology with this kind of ignorance. Sacrifice a goat? Sign over your firstborn? Oh boy...

**Reply by AdrianAtGoogle** [reply]
Your frustration is communicated, SGECZ.  The SV team is still spinning up on github and fitting the information hose into our process.  It looks like the Google App release with the fix for the early placement issue has not hit the play store yet, and goat sacrifices would only complicate the meantime, not shorten it.

**Reply by david-rhodes** [reply]
Thank you for the update @AdrianAtGoogle! We do appreciate the active communication.

**Reply by SGECZ** [reply]
> 
> 
> Your frustration is communicated, SGECZ. The SV team is still spinning up on github and fitting the information hose into our process. It looks like the Google App release with the fix for the early placement issue has not hit the play store yet, and goat sacrifices would only complicate the meantime, not shorten it.

ok, no goats then.
but yes, thank you for the update on this. ;)

Cheers, Stefan

**Reply by SGECZ** [reply]
Just updated the Google app - issue seems fixed. No goat was harmed in the process. Thank you everyone for your contribution!
